cmd/build.go

```diff
@@ -43,12 +44,17 @@ const clangBinName = "o32-clang"
 var crossCompile = false
 var engineCachePath string
 
+var buildTargetDefault = "lib/main_desktop.dart"
+var buildBranchDefault = ""
+var buildCachePathDefault = ""
+var buildOpenGlVersionDefault = "3.3"
+
```

Comment:
can be `const`
and should be placed in the internal/config/config.go file

Reply:
Can this be placed in internal/config/config.go file?

Reply:
Yes